api/python/provisioner/ui/main_menu.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+#
+import curses
+import config
+import importlib
+from window import Window
+from color_code import ColorCode
+
+
+class MainMenu(Window):
+    _menu = None
+    _menu_dict = None
+
+    def __init__(self, window):
+        super().__init__(window)
+        self.set_menus(config.menu)
+        self._menu_dict = config.menu
+        self._menu.append("Exit")
+
+    def get_menu(self):
+        return self._menu
+
+    def set_menus(self, menu):
+        self._menu_dict = menu
+        tmp_menu = list(menu.keys())
+        self._menu = tmp_menu
+
+    def create_window(self, **kwargs):
+        color_code = kwargs['color_code']
+        selected_rows = kwargs['menu_code']
+        self._window.border()
+        col_code_attr = ColorCode.get_color_pair(color_code)
+        self.create_menu_head()
+
+        for idx, row in enumerate(self._menu):
+            x = self.get_max_width() // 2 - len(row) // 2
+            y = self.get_max_height() // 2 - len(self._menu) // 2 + idx
+
+            if idx == selected_rows:
+                self.on_attr(col_code_attr)
+                self._window.addstr(y, x-3, ">> ")
```

Comment:
hard codes plus missed spaces around binary operator

Reply:
comment is addressed